fault-proof/tests/e2e.rs

```diff
@@ -453,7 +453,9 @@ mod e2e {
                 let new_game_info = factory.gameAtIndex(new_game_index).call().await?;
 
                 // Check that the new game doesn't build on the invalid chain
-                let new_game = env.fault_dispute_game(new_game_info.proxy_).await?;
+                let new_game = env
+                    .fault_dispute_game_with_role(new_game_info.proxy_, common::env::Role::Proposer)
```

Comment:
```suggestion
                    .fault_dispute_game(new_game_info.proxy_)
```

Please update this in some other places too, before we move on.

Reply:
7a41544e0ffda665132a5b6e948dc6bb2871e567